xarray/core/dataarray.py

```diff
@@ -1377,13 +1378,15 @@ def func(self, *args, **kwargs):
         return func
 
     @staticmethod
-    def _binary_op(f, reflexive=False, join='inner', **ignored_kwargs):
+    def _binary_op(f, reflexive=False, join=None, **ignored_kwargs):
         @functools.wraps(f)
         def func(self, other):
             if isinstance(other, (Dataset, groupby.GroupBy)):
                 return NotImplemented
             if hasattr(other, 'indexes'):
-                self, other = align(self, other, join=join, copy=False)
+                # if user does not specify join, default to OPTIONS['join']
+                how_to_join = join or OPTIONS['join']
```

Comment:
Funny, your suggestion is the same as my first edit to the code, but then my teammates suggested I should use the other options.

Nomenclature is not my forte.  I'll make the default
```
OPTIONS = {'display_width': 80, 'align_type': "inner"}
```
and change your suggestion to the following
```
align_type = OPTIONS['align_type'] if join is None else join
self, other = align(self, other, join=align_type, copy=False)
```
if there's no objection.